src/plugins/core/range.ts

```diff
@@ -329,8 +329,9 @@ export class RangeAdapter implements CommandHandler<CoreCommand> {
    *
    * @param range the range (received from getRangeFromXC or getRangeFromZone)
    * @param forSheetId the id of the sheet where the range string is supposed to be used.
+   * @param [useFixedZone=false] if true, the range will be returned with fixed row and column
    */
-  getRangeString(range: Range, forSheetId: UID): string {
+  getRangeString(range: Range, forSheetId: UID, useFixedZone: boolean = false): string {
```

Comment:
what about a named parameter instead of simple boolean.
It looks more readable.

```ts
this.getters.getRangeString(range, sheetId, { useFixedReferences: true })
// vs
this.getters.getRangeString(range, sheetId, true)
```